config/rbac/auth_proxy_service.yaml

```diff
@@ -14,7 +14,7 @@ metadata:
 spec:
   ports:
   - name: https
-    port: 8443
+    port: 8080
```

Comment:
Why do we need to change this? 8080 would suggest http and this is still https no?

Reply:
That slipped past me when changing the value.
I changed to 8080 as that's the metrics port.

I think a separate port called `metrics`, on port 8080.
Probably a separate Service as well?
I'm not sure what this auth proxy service is for

Reply:
The auth proxy is specifically for protecting the metrics endpoint, and it looks like we have removed it in other operators as it's not fully compatible with the OpenShift monitoring stack and other reasons see https://github.com/Kuadrant/kuadrant-operator/pull/33

Id say we should follow suit and remove it from this repo also. Can be done in a follow up task though i think.

Reply:
Created a follow on issue to remove it  https://github.com/Kuadrant/multicluster-gateway-controller/issues/411